src/emulator/functionsEmulatorShared.ts

```diff
@@ -282,3 +282,11 @@ export function findModuleRoot(moduleName: string, filepath: string): string {
 
   return "";
 }
+
+export function formatHost(info: { host: string; port: number }): string {
+  if (info.host.includes(":")) {
+    return `[${info.host}]:${info.port}`;
```

Comment:
This admittedly isn't my corner of the codebase, but I have no idea what this is or what this means.  Does it merit a comment?